src/panels/lovelace/components/hui-image.ts

```diff
@@ -223,6 +225,7 @@ export class HuiImage extends LitElement {
           : html`
               <img
                 id="image"
+                alt=${this.altText || ""}
```

Comment:
Let's rename to `imageAlt` and use `ifDefined(this.imageAlt)` instead.  The default should just be to omit `alt` if omitted from the element, not to hide it.

Reply:
On 2nd though, this should be reverted as well since it was for picture entity.